sycl/test/esimd/simd_copy_to_copy_from.cpp

```diff
@@ -34,25 +34,23 @@ SYCL_EXTERNAL void kernel2(int *ptr) SYCL_ESIMD_FUNCTION {
   v0.copy_to(ptr);
 }
 
-// --- Negative tests.
-
-// Incompatible target.
 SYCL_EXTERNAL void kernel3(local_accessor<int, 1> &buf) SYCL_ESIMD_FUNCTION {
   simd<int, 32> v1(0, 1);
   simd<int, 32> v0;
-  // CHECK: simd_copy_to_copy_from.cpp:44{{.*}}error: no matching member function for call to 'copy_from'
```

Comment:
Do we still need kernel3? After you removed the CHECK of fail for local accessor it doesn't mean the error is not printed, right? Without CHECK here kernel3 doesn't make sense to me.

Reply:
No, it did not became a positive test. With clang++ run returning an error code and no CHECK for this function you can never tell if the subtest-test kernel3 passes or fails.

For example, if kernel3 would report an error, the test test would just not detect it because there is no CHECK now.

Reply:
We can just remove kernel3 now.

Reply:
Ok, I see now, that the test uses --implicit-check-not="error:", which does the trick. Thank you.